manifests/config/puppetserver.pp

```diff
@@ -4,7 +4,11 @@
 ) {
   require ::puppetserver::augeas
 
-  $targetdir = '/etc/puppetserver/conf.d'
+  if versioncmp($::puppetversion, '4.0.0') >= 0 {
```

Comment:
@raphink Well, see my initial comment in the PR.  
It's not linked, but I don't know a better way to make the decision here, there is no $::puppetserverversion...